### PR TITLE
Use gem toml-rb instead of toml

### LINKED ISSUE #63: Use gem 'toml-rb' instead of 'toml'
Development of the `toml` gem has not been active in a while, despite bugs (see https://github.com/jm/toml/issues/41).

I've had good experiences with [toml-rb](https://github.com/eMancu/toml-rb). The latest release was on 6/12. Have you considered using it instead?

### PR DESCRIPTION
**Pull request by alvaromorales**
Fixes #63

### CODE REVIEW

File: resources/config.rb [hunk 1]
```diff
@@ -23,10 +23,10 @@
 property :config, Hash, required: true
 
 action :create do
-  require 'toml'
+  require 'toml-rb'
```
Comment (fishnix) [inline resources/config.rb]: I think this is wrong - this gem should still `require toml` which is kind of annoying.

Comment (alvaromorales) [inline resources/config.rb]: you're right! fixed

**Comment by fishnix** [comment]
I think @bdangit is working on getting wercker going.  Looks good to me :+1:

**Comment by bdangit** [comment]
yep working on #95. with that in mind, if you can add in chefspec and at least validate via kitchen then we can merge this in.

**Comment by bdangit** [comment]
Just ran through test-kitchen against 12.5 chef-clients for ubuntu, debian, and centos.  They all passed.  I can only imagine that they will work for almost all thats stated in `.kitchen.yml`

Will merge once #95 is in.

**Comment by bdangit** [comment]
going to merge this despite the error that wercker states.

**Pull request merged**